source/JsMaterialX/test/document.spec.js

```diff
@@ -89,16 +89,16 @@ describe('Build Document', () => {
         material = doc.addMaterial();
         shaderRef = material.addShaderRef('shaderRef1', 'simpleSrf');
         expect(material.getPrimaryShaderName()).to.equal('simpleSrf');
-        expect(material.getPrimaryShaderParameters().length).to.equal(1);
-        expect(material.getPrimaryShaderInputs().length).to.equal(2);
+        expect(material.getPrimaryShaderParameters().length).to.equal(0);
```

Comment:
Should this call be removed, does getPrimaryShaderParameters still exists?

Reply:
I haven't done much here since this is for the Material Element class which needs a second pass to actually remove it. Minimizing churn for this PR.